website/blog/2025-12-04-release-1.24.md

```diff
@@ -0,0 +1,257 @@
+---
+title: Podman Desktop 1.24 Release
+description: Podman Desktop 1.24 has been released!
+slug: podman-desktop-release-1.24
+authors: [simonrey1]
+tags: [podman-desktop, release, kubernetes, openshift]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.23.png
+---
+
+import ThemedImage from '@theme/ThemedImage';
+
+Podman Desktop 1.24 Release! 🎉
+
+![podman-desktop-hero-1.24](/img/blog/podman-desktop-release-1.23/banner.png)
+
+Podman Desktop 1.24 is now available. [Click here to download it](/downloads)!
+
+This release brings exciting new features and improvements.
+
+## Release details
```

Comment:
I think we are missing the little section that has a 1-line intro. This is also what shows up in the app so super important!

Reply:
I added 3 bullet points for the intro, see commit https://github.com/podman-desktop/podman-desktop/pull/15223/commits/b19aaf2d72aec9b3c08e124e0ac21e4e30d79683